backend/src/utils/mq.rs

```diff
@@ -136,6 +137,9 @@ lazy_static! {
         };
         url
     };
+    static ref BACKEND_TEST_MODE: bool =
+        // **TODO**: set default to 'false' later
+        env::var("BACKEND_TEST_MODE").map(|x| x.parse::<bool>().unwrap()).ok().unwrap_or(true);
```

Comment:
When you uncomment the lines in backend-test.yml, you can change the default value to false.